src/Core/Language/Pack/Loader/RemoteLanguagePackLoader.php

```diff
@@ -24,6 +24,8 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 
+declare(strict_types=1);
```

Comment:
I agree with @matks however since the PR already has BC breaks I think it's the occasion to integrate this one It can be noted in the PR's description at worst